jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/list/AdvancedSearchFiltersViewImpl.java

```diff
@@ -220,11 +224,11 @@ protected void onDateValueChange(final String label,
         final DateRange dateRange = new DateRange(fromDateValue,
                                                   toDateValue);
         addActiveFilter(label,
-                        translationService.format(Constants.FROM)
+                        constants.From()
```

Comment:
@rbarriuso FYI, somehow GWT compiler did not like mixing both GWT and Errai i18n in the same interface.